tools/donate_cpu_lib.py

```diff
@@ -249,12 +255,18 @@ def run_command(cmd):
     print(cmd)
     startTime = time.time()
     p = subprocess.Popen(cmd.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    comm = p.communicate()
+    return_code = RETURN_CODE_TIMEOUT
```

Comment:
nit; I think `return_code = RETURN_CODE_TIMEOUT` could be moved into the except below.